roles/engine/tasks/copy_binary.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Copy binary and shared library files
+  ansible.builtin.copy:
```

Comment:
I think we should the build process here as well, most of the times there is just engine update